tests/test_api.py

```diff
@@ -896,9 +896,11 @@ def test_config_cache_size(app_client_larger_cache_size):
 
 def test_config_force_https_urls():
     with make_app_client(settings={"force_https_urls": True}) as client:
-        response = client.get("/fixtures/facetable.json?_size=3&_facet=state")
+        response = client.get(
+            "/fixtures/facetable.json?_size=3&_facet=state&_extra=next_url,suggested_facets"
+        )
         assert response.json["next_url"].startswith("https://")
-        assert response.json["facet_results"]["state"]["results"][0][
+        assert response.json["facet_results"]["results"]["state"]["results"][0][
```

Comment:
A reminder that I changed the shape of the `facet_results` dictionary to have `results` and `timed_out` keys.